web-components/src/components/PrefinanceTag/PrefinanceTag.tsx

```diff
@@ -0,0 +1,33 @@
+import { cn } from 'web-components/src/utils/styles/cn';
+
+import { PREFINANCE } from '../cards/ProjectCard/ProjectCard.constants';
+import { PrefinanceIcon } from '../icons/PrefinanceIcon';
+import { Label } from '../typography';
+
+export const PrefinanceTag = ({
```

Comment:
on project card, now it looks slightly different than before (icon/font size too large, wrong position).
On your branch:
![image](https://github.com/user-attachments/assets/d2b10012-beb2-4992-960d-702752f8d9be)

VS on dev:
![image](https://github.com/user-attachments/assets/1fce2e19-94bc-4b6e-b1e8-468e45ba8fa4)

Reply:
Same on the top of the project page:
Your branch:
![image](https://github.com/user-attachments/assets/d5808e81-cd7e-426e-b7a1-c2ba03234e4c)

VS dev:
![image](https://github.com/user-attachments/assets/4d5f9816-a05f-47de-b36e-9736091cab33)

Reply:
see initial design: https://www.figma.com/design/EPSVuGURedvRfO6LxUAQZs/Pre-financing?node-id=2-35410&t=rLt910pE0aeb38kE-0